coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
Yes. Comparing file contents won't let user create a fork of the angular app and just keep the json configs. But that would mean they can't use the latest changes in angular app. What do we do?